src/contracts/token/Eigen.sol

```diff
@@ -134,11 +134,6 @@ contract Eigen is OwnableUpgradeable, ERC20VotesUpgradeable {
         require(bEIGEN.transfer(msg.sender, amount), "Eigen.unwrap: bEIGEN transfer failed");
```

Comment:
can we delete the mint function

Reply:
I have been aiming for minimal code changes. I think this might complicate existing tests as well :/

Reply:
resolved this offline, keeping code changes minimal here